README.md

```diff
@@ -1,2 +1,64 @@
-# First-capstone-Project
-This project shows my acquired knowledge so far on Html, Css and Javascript
+# First-capstone-Project: Milestone 6
+
+This project shows my acquired knowledge so far;
+
+> The capstone project demonstrates Javascript, Html and Css.
+> This is also a project that shows my proficiency as I go through Microverse. It is a work-in-progress and I hope to make it a masterpiece.
+
+## Live Demo
+
+[Link to Live Demo](https://eternalgratis.github.io/First-capstone-Project/)
+
+## Built With
+
+- HTML
+- CSS
+- Linter
+- Javascript
+
+## Authors
+
+👤 **Titilope Apuabi**
+
+- GitHub: [@eternalgratis](https://github.com/Eternalgratis)
+- Twitter: [@eternalgratis](https://twitter.com/eternalgratis)
+- LinkedIn: [@eternalgratis](https://www.linkedin.com/in/titilope-apuabi-69a98719b/)
+
+### Collaborators
+
+👤 **Diego Luna Granados** _(Week 4: Mobile Menu & Popup window)_
+
+- GitHub: [@SlowKingV](https://github.com/SlowKingV)
+- Twitter: [@SlowKingVI](https://twitter.com/SlowKingVI)
+- LinkedIn: [Diego Luna Granados](https://www.linkedin.com/in/diego-luna-granados/)
+
+👤 **Diego Luna Granados** _(Week 4: Mobile version nav button)_
+
+- GitHub: [@SlowKingV](https://github.com/SlowKingV)
+- Twitter: [@SlowKingVI](https://twitter.com/SlowKingVI)
+- LinkedIn: [Diego Luna Granados](https://www.linkedin.com/in/diego-luna-granados/)
```

Comment:
- [ ] This Is a solo project so you do not need the rest of the authors, kindly remove them